doc/sphinx-guides/source/container/running/demo.rst

```diff
@@ -0,0 +1,132 @@
+
+We plan to ship a "demo" persona but it is not ready yet. See also :ref:`configbaker-personas`.
+
+Common Operations
+-----------------
+
+Starting the Containers
++++++++++++++++++++++++
+
+First, download :download:`compose.yml <../../../../../docker/compose/demo/compose.yml>` and place it somewhere you'll remember.
+
+Then, run ``docker compose up`` in the directory where you put ``compose.yml``
+
+Starting the containers for the first time involves a bootstrap process. You should see "have a nice day" output at the end.
+
+Stopping the Containers
++++++++++++++++++++++++
+
+You might want to stop the containers if you aren't using them. Hit ``Ctrl-c`` (hold down the ``Ctrl`` key and then hit the ``c`` key).
+
+You data is still intact and you can start the containers again with ``docker compose up``.
+
+Deleting the Containers
++++++++++++++++++++++++
+
+If you no longer need the containers because your demo or evaluation is finished and you want to reclaim disk space, run ``docker compose down`` in the directory where you put ``compose.yml``.
+
+Deleting the Data Directory
++++++++++++++++++++++++++++
+
+Data related to the Dataverse containers is placed in a directory called ``docker-dev-volumes`` next to the ``compose.yml`` file. If you are finished with your demo or evaluation or you want to start fresh, simply delete this directory.
+
+Configuration
+-------------
+
+Configuration is described in greater detail under :doc:`/installation/config` in the Installation Guide, but there are some specifics to running in containers you should know about.
+
+.. _configbaker-personas:
+
+Personas
+++++++++
+
+When the containers are bootstrapped, the "dev" persona is used. In the future we plan to add a "demo" persona that is more suited to demo and evaluation use cases.
+
+Database Settings
++++++++++++++++++
+
+Updating database settings is the same as described under :ref:`database-settings` in the Installation Guide.
+
+MPCONFIG Options
+++++++++++++++++
+
+The compose file contains an ``environment`` section with various MicroProfile Config (MPCONFIG) options. You can experiment with this by adding ``DATAVERSE_VERSION: foobar`` to change the (displayed) version of Dataverse to "foobar".
+
+JVM Options
++++++++++++
+
+JVM options are not especially easy to change in the container. The general process is to get a shell on the "dataverse" container, change the settings, and then stop and start the containers. See :ref:`jvm-options` for more.
+
+Troubleshooting
+---------------
+
+Bootstrapping Did Not Complete
+++++++++++++++++++++++++++++++
+
+In the compose file, try increasing the timeout in the bootstrap container by adding something like this:
+
+.. code-block:: bash
+
+   environment:
+     - TIMEOUT=10m
```

Comment:
If we're worried about this, should we preemptively put this in the compose file?

Reply:
Yes, and document it prominently. I had some failed startups and it took me a while to realize that the bootstrap did not run due the short timeout.